packages/fluentui/e2e/tests/popupClickHandling-example.tsx

```diff
@@ -15,16 +15,18 @@ const PopupClickHandlingExample = () => {
         <Button id={selectors.triggerButtonId} content="Open Popup" icon="expand" onClick={() => setShow(true)} />
       }
       content={
-        !!show && (
-          <Button
-            id={selectors.popupContentButtonId}
-            content="click me"
-            onClick={e => {
-              e.preventDefault();
-              setShow(!show);
-            }}
-          />
-        )
+        <div>
```

Comment:
Since `popupContent` will be null when `false` is passed to `content` there would be no content wrapper in this case, what makes sense for me. To achieve the result expected in the test I am adding explicit the wrapper

Reply:
I think that then we'd need to describe this in Breaking changes, because as you show in the test, it would require users to upgrade their code, for the example to still work the same

Reply:
Sure thing, makes sense for me. I will just wait @layershifter to comment if this is the proper fix and will add it on the changelog